x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression/predicate/operator/arithmetic/Sub.java

```diff
@@ -27,4 +27,9 @@ public Sub(Source source, Expression left, Expression right) {
     protected Sub replaceChildren(Expression newLeft, Expression newRight) {
         return new Sub(source(), newLeft, newRight);
     }
+
+    @Override
+    public ArithmeticOperation inverse(Source source, Expression left, Expression right) {
```

Comment:
return `Add` instead of `ArithmeticOperation`

Reply:
Fixed.